test/new_device_startmode_test.rb

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+require "yast"
+
+Yast.import "ProductFeatures"
+Yast.import "LanItems"
+
+describe "LanItemsClass#new_device_startmode" do
```

Comment:
this is wrong as this method is no longer in network-ng, or am I wrong?

Reply:
Yes, you are right. Dropped.